plugins/participation/plugin.go

```diff
@@ -141,7 +143,7 @@ func provide(c *dig.Container) {
 }
 
 func configure() {
-	restapiv1.AddFeature(Plugin.Name)
+	restapiv2.AddPlugin("participation")
```

Comment:
Beginning with stardust, should we just version all plugins?
```suggestion
	restapiv2.AddPlugin("participation/v1")
```